src/loader/objects/local-door.hpp

```diff
@@ -3,11 +3,16 @@
 #include <cstdio>  // FILE
 
 #include "level/local-door.hpp"
-#include "loader/load-object.hpp"
+#include "loader/load-object.tpp"
 
 namespace loader {
     class LocalDoor : public LoadObject<level::LocalDoor> {
       public:
+        /**
+         * @brief carica gli oggetti per i door locali dal file
+         *
+         * @param file File descriptor da cui caricare i door locali

```

Comment:
anche in queste classi credo sia meglio commentare con semplicemente implementazione di loadobject per la local door, poi quello che fa lo spieghi in load objec